### DISCUSSION TITLE
ERROR: regarding useLocalStorageState -  is not a function or its return value is not iterable

### DISCUSSION THREAD

**Discussion by NicoleHoerner** [neuefische/questions#729] [Web] [answered]
Hello,
I'm working on my addFlight function, implementing useLocalStorageState to store data of added flight and I get an error message that I don't understand. It seems to be related to useLocalStorageState.

ERROR

Server Error
TypeError: (0 , use_local_storage_state__WEBPACK_IMPORTED_MODULE_6__.useLocalStorageState) is not a function or its return value is not iterable

This error happened while generating the page. Any console logs will be displayed in the terminal window.
<img width="1320" alt="Bildschirmfoto 2023-09-13 um 02 46 58" src="https://github.com/neuefische/questions/assets/128461153/f0956fa6-c524-4dea-9bc9-71b19211f412">

What have I already tried:

- I have already searched for this error online and found solutions that didn't help me.

E.g. update an old React version.

- I have executed:

npm install use-local-storage-state
and found under dependencies this version: "use-local-storage-state": "^19.0.4"
this is the latest version on the npm documentation.
I have checked the import statement.

Thanks for any help!
Regards, Nicole

**Comment by Kheirah** [comment, marked as answer]
The import is a default, not a named. The error pops up because there is no named export in `use-local-storage-state` called `useLocalStorageState `. Drop the braces like this and it'll work.
```js
import useLocalStorageState from 'use-local-storage-state'
```
